### DISCUSSION TITLE
Convert to issue should allow selecting an issue template

### DISCUSSION THREAD

**Discussion by Arcanorum** [community/community#20762] [Projects and Issues]
Currently for an item on project board when you click "Convert to issue" it just creates a new blank issue.

Many projects have templates set up for how they want each kind of issue to be formatted.

There should be a way to select a template to use when creating an issue in this way.

![image](https://user-images.githubusercontent.com/9785268/179405463-b766c052-d7f6-4b51-b32b-f0c6139babb7.png)

Should allow selecting one of these

![image](https://user-images.githubusercontent.com/9785268/179405528-e38fd92e-5ed8-4654-8235-71e95e4abfb4.png)

Otherwise you need to go into the issue template and copy/paste the contents each time...

**Comment by ferdnyc** [comment]
@Arcanorum 

The problem I see with that is, "Convert to issue" is an _unattended_ conversion of an existing Draft Issue's content from the project board over to a repo's Issues list. Issue Templates are for _newly created_ issues. With "Convert to issue", the issue is created automatically (without editing) using the existing title and description as its content. Those would have to be thrown away to use the Issue Template.

Now, what might be interesting is to be able to define Issue Templates / Issue Forms _**for**_ Projects, so that you can choose a template to fill out when first creating the Draft Issue, just like you can when filing a new issue in a repository.

But if you want to use a certain repository's New Issue template(s) when creating an issue, the best way to do that is to first create the issue **IN** the repository, then add the issue to the Project — rather than the other way around.

**Reply by joaomas** [reply]
Yes, @Arcanorum is a good explanation of why there isn't a possibility to convert draft witch issue template.

But the problem in your alternative solution is that draft is draft and issue is issue, that is, create issue thinking like draft, for me, isn't make sense... will have a lot of issue garbage.

Thanks.

**Comment by MrChocolateMoose** [comment]
I agree with @Arcanorum.  This request is a basic "common" sense thing. 

The current behavior makes the "convert to issue" feature mostly useless and encourages stub issues without content that don't have acceptance criteria / structure.

Does anybody know if this functionality was improved in the "list" beta?
